schemas/metaschema/domains.schema

```diff
@@ -66,4 +67,19 @@
   Volume: { type: 'number', unit: 'Litre', symbol: 'l' },
   Speed: { type: 'number', unit: 'Metres per second', symbol: 'm/s' },
   Ratio: { type: 'number' },
+  ActionType: Enum(
+    'Create', 'Update', 'Delete', 'ShowDetail',
+    'Digicert', 'Steps', 'Custom'
+  ),
+  // determines how Action with multiple items will be contained
+  // Each - commit after each item, All - commit after all
+  TransactionMode: Enum('Each', 'All'),
```

Comment:
```suggestion
  TransactionMode: Enum('Single', 'All'),
```

Reply:
`All` by default. By the way if `Action` is `Sequential` or `Parallel` we will disallow to return `Execute` for multiple entities.